Swiften/Client/Client.cpp

```diff
@@ -148,13 +154,27 @@ void Client::handleConnected() {
     discoManager->handleConnected();
 }
 
+void Client::requestFeatures() {
```

Comment:
I'm not sure this happening automatically is consistent with the design elsewhere, where network traffic isn't generally generated without being requested. It might be sensible, but it's not plugged in anywhere else, which means it's going to end up duplicated by the current disco logic in Swift isn't it?